packages/simcore-sdk/src/simcore_sdk/node_ports_common/filemanager.py

```diff
@@ -381,23 +446,14 @@ async def entry_exists(
 ) -> bool:
     """Returns True if metadata for s3_object is present"""
     try:
-        async with ClientSessionContextManager(client_session) as session:
-            log.debug("Will request metadata for s3_object=%s", s3_object)
-
-            file_metadata: FileMetaDataGet = await storage_client.get_file_metadata(
-                session=session,
-                file_id=s3_object,
-                location_id=store_id,
-                user_id=user_id,
-            )
-            log.debug(
-                "Result for metadata s3_object=%s, result=%s",
-                s3_object,
-                f"{file_metadata=}",
-            )
-            return bool(file_metadata.file_id == s3_object)
+        file_metadata: FileMetaDataGet = await _get_file_meta_data(
+            user_id, store_id, s3_object, client_session
+        )
+        return bool(file_metadata.file_id == s3_object)

```

Comment:
I do not think you need to cast this to bool. the `==` operator returns a bool

Reply:
transforming to strings might be the right call